app/controllers/songs_controller.rb

```diff
@@ -1,5 +1,34 @@
 class SongsController < ApplicationController
   def index
+    respond_to do |format|
```

Comment:
I'm trying to understand the design and usage of the jQeury database gem. Correct me if I'm wrong: on the initial page load, Rails will not send any song data to the browser. But the JS on the page will get triggered and send a GET request and hit this code, which will then return all the songs? So the song data is never directly sent with the HTML body but retrieved when the gem requests it?